src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vmss/azuredeploy.json

```diff
@@ -461,28 +491,26 @@
     "vnetTemplateUri": "https://azuresdkci.blob.core.windows.net/templatehost/CreateVnet_2016-07-18/azuredeploy.json",
     "vnetTemplateUriEmpty": "https://azuresdkci.blob.core.windows.net/templatehost/CreateVnet_2016-07-18/azuredeploy_empty.json",
     "vnetTemplateFilePaths": {
-      "existing": "[variables('vnetTemplateUriEmpty')]",
+      "existingId": "[variables('vnetTemplateUriEmpty')]",
+      "existingName": "[variables('vnetTemplateUriEmpty')]",
       "new": "[variables('vnetTemplateUri')]"
     },
     "publicIpTemplateUri": "https://azuresdkci.blob.core.windows.net/templatehost/CreatePublicIp_2016-07-19/azuredeploy.json",
     "publicIpTemplateUriEmpty": "https://azuresdkci.blob.core.windows.net/templatehost/CreatePublicIp_2016-07-19/azuredeploy_empty.json",
     "ipTemplateFilePaths": {
+      "existingId": "[variables('publicIpTemplateUriEmpty')]",
       "existingName": "[variables('publicIpTemplateUriEmpty')]",
       "new": "[variables('publicIpTemplateUri')]",
       "none": "[variables('publicIpTemplateUriEmpty')]"
     },
     "lbTemplateUri": "https://azuresdkci.blob.core.windows.net/templatehost/CreateLb_2016-07-19/azuredeploy.json",
     "lbTemplateUriEmpty": "https://azuresdkci.blob.core.windows.net/templatehost/CreateLb_2016-07-19/azuredeploy_empty.json",
     "lbTemplateFilePaths": {
-      "existing": "[variables('lbTemplateUriEmpty')]",
+      "existingId": "[variables('lbTemplateUriEmpty')]",
```

Comment:
see previous comment, we have extra functionality in the VMSS case

---
In reply to: [73383556](https://github.com/Azure/azure-cli/pull/599#discussion_r73383556) [](ancestors = 73383556)